test/conf/logback-burntest.xml

```diff
@@ -20,7 +20,7 @@
   <define name="instance_id" class="org.apache.cassandra.distributed.impl.InstanceIDDefiner" />
 
   <!-- Shutdown hook ensures that async appender flushes -->
-  <shutdownHook class="ch.qos.logback.core.hook.DelayingShutdownHook"/>
+  <shutdownHook class="ch.qos.logback.core.hook.DefaultShutdownHook"/>
```

Comment:
renamed in logback in:
https://github.com/qos-ch/logback/commit/4d705d9045b4df494b358984d326efb2ee9d6bd0